### DISCUSSION TITLE
addReducedDims

### DISCUSSION THREAD

**Discussion by bjstewart1** [GreenleafLab/ArchR#1879] [Feature Requests]
It may well be very useful to write an "add-er" to addReducedDims that are not LSI to projects.
For example if users wish to use the (very useful) `peakVI` or `multiVI` tools in `scvi-tools`, these latent representations could be added to archR project. 
I cannot see that this is possible at the moment.

**Comment by rcorces** [comment]
This is a good suggestion. What format would the input be in? a matrix of cells by dimensions? Is it reasonable to assume that all cells would be represented in the dim. reduction? Would you be willing to test this functionality if implemented?

**Comment by bjstewart1** [comment]
yes I would interested in testing this functionality. I have been using a basically functional couple of lines of code to do this, but it would be great if it were a formally integrated function for the package.
The input would be a matrix of cells by dims and I think it would entirely reasonable to assume all cells would be representated in any reduced dimensionality representation passed to the function.

**Comment by james-cranley** [comment]
@rcorces i think this would be a good feature. for instance I would like to run p2g linkage and supply a peakVI space which has been batch corrected.
